src/resources/assets/js/common.js

```diff
@@ -1,47 +1,12 @@
-// To make Pace works on Ajax calls
-$(document).ajaxStart(function() { Pace.restart(); });
-
-// polyfill for `startsWith` from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
-if (!String.prototype.startsWith) {
-    Object.defineProperty(String.prototype, 'startsWith', {
-        value: function(search, rawPos) {
-            var pos = rawPos > 0 ? rawPos|0 : 0;
-            return this.substring(pos, pos + search.length) === search;
-        }
-    });
-}
-
-
-
-// polyfill for entries and keys from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#polyfill
-if (!Object.keys) Object.keys = function(o) {
-    if (o !== Object(o))
-        throw new TypeError('Object.keys called on a non-object');
-    var k=[],p;
-    for (p in o) if (Object.prototype.hasOwnProperty.call(o,p)) k.push(p);
-    return k;
-}
-
-if (!Object.entries) {
-    Object.entries = function( obj ){
-        var ownProps = Object.keys( obj ),
-            i = ownProps.length,
-            resArray = new Array(i); // preallocate the Array
-        while (i--)
-            resArray[i] = [ownProps[i], obj[ownProps[i]]];
-        return resArray;
-    };
-}
-
 // Ajax calls should always have the CSRF token attached to them, otherwise they won't work
 $.ajaxSetup({
     headers: {
-        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        'X-CSRF-TOKEN': document.querySelector('meta[name="csrf-token"]').content,
     }
 });
 
 // Enable deep link to tab
-document.querySelectorAll('.nav-tabs a').forEach(function (elem) {
-    if(elem.dataset.name === location.hash.substr(1)) (new bootstrap.Tab(elem)).show();
-    elem.addEventListener('click', () => location.hash = elem.dataset.name);
-});
```

Comment:
`substr` is deprecated;
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr